annotation-utils/src/main/java/edu/isi/vista/annotationutils/SerializedCasConverter.java

```
@@ -0,0 +1,111 @@
+
+        CTEventSpan event = type.getGovernor();
+        String role = type.getRelation_type();
+        CTEventSpan argument = type.getDependent();
+
+        if (!eventArguments.containsKey(event)) {
+          eventArguments.put(event, new ArrayList<>());
+        }
+        eventArguments.get(event).add(new Argument(role, argument.getBegin(), argument.getEnd()));
+      }
+
+      List<CTEvent> events = new ArrayList<>();
+      for (Map.Entry<CTEventSpan, List<Argument>> entry : eventArguments.entrySet()) {
+        // create and add event
+        CTEventSpan eventAnnotation = entry.getKey();
+        events.add(
+            new CTEvent(eventAnnotation.getBegin(), eventAnnotation.getEnd(), entry.getValue()));
+
+      }
+
+      ObjectMapper mapper = new ObjectMapper();
+      try (OutputStream os = new FileOutputStream(output)) {
+        mapper.writeValue(os, events);
+        log.info("Exported to " + output.getAbsolutePath());
+      }
+
+    } catch (IOException | UIMAException e) {
+      log.error("Caught exception {}", e);
+      System.exit(1);
+    }
+  }
+
+  private static CAS readCasFromFile(File serializedCasFile) throws IOException, UIMAException {
+    if (!serializedCasFile.exists()) {
+      throw new FileNotFoundException(serializedCasFile + " not found");
+    }
+
+    CAS cas = CasCreationUtils.createCas((TypeSystemDescription) null, null, null);
+    readSerializedCas(cas, serializedCasFile);
+
+    return cas;
+  }
+
+  private static void readSerializedCas(CAS aCas, File aFile) throws IOException {
+    try (ObjectInputStream is = new ObjectInputStream(new FileInputStream(aFile))) {
+      CASCompleteSerializer serializer = (CASCompleteSerializer) is.readObject();
+      deserializeCASComplete(serializer, (CASImpl) aCas);
+      // Initialize the JCas sub-system which is the most often used API in DKPro Core
+      // components
+      aCas.getJCas();
+    } catch (ClassNotFoundException e) {
```

If not checked, let it bubble up as well

---

Moved exception handling to top level.